code_gen/api/khr_material_pbr.json

```diff
@@ -0,0 +1,139 @@
+{
+    "info" : {
+        "name" : "KHR_MATERIAL_PBR",
+        "type" : "feature",
+        "dependencies" : []
+    },
+
+    "objects" : [
+        {
+            "type" : "ANARI_MATERIAL",
+            "name" : "pbr",
+            "description" : "pbr material object",
+            "parameters" : [
+                {
+                    "name" : "name",
+                    "types" : ["ANARI_STRING"],
+                    "tags" : [],
+                    "description" : "optional object name"
+                }, {
+                    "name" : "baseColor",
+                    "types" : ["ANARI_FLOAT32_VEC3", "ANARI_SAMPLER", "ANARI_STRING"],
```

Comment:
This should probably also allow `ANARI_FLOAT32_VEC4` since it also handles samplers and attributes with alpha channel

Reply:
The color itself is `VEC3`, which does not rule out that the texture / sampler can have a 4th channel (we did not do that as well for `Matte`). To set alpha a a scalar just use the `opacity` parameter.